python/ptest/cases/dcdc_checkout_case.py

```diff
@@ -0,0 +1,100 @@
+from .base import SingleSatOnlyCase
+
+class DCDCCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("DCDC cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state("pan.cycle_no")
+
+        adcs_cmd = simulation.flight_controller.read_state("dcdc.ADCSMotor_cmd")
+        sph_cmd = simulation.flight_controller.read_state("dcdc.SpikeDock_cmd")
+        disable_cmd = simulation.flight_controller.read_state("dcdc.disable_cmd")
+        reset_cmd = simulation.flight_controller.read_state("dcdc.reset_cmd")
+        adcs_pin = simulation.flight_controller.read_state("dcdc.ADCSMotor")
+        sph_pin = simulation.flight_controller.read_state("dcdc.SpikeDock")
```

Comment:
The naming is a little confusing here; the "_pin" suffix to me implies that you're reading a pin # off the device, which is not actually what's happening. I suggest "adcs_rd".